doc/rst/usingchapel/chplenv.rst

```diff
@@ -650,28 +694,20 @@ CHPL_LLVM
        none           do not support llvm/clang-related features
        ============== ======================================================
 
-   .. (comment) -minimal can be used but is only interesting for developers
-       llvm-minimal   as above, but only build and link LLVM ADTs
-       system-minimal as above, but only link LLVM ADTs
-
-   If unset, ``CHPL_LLVM`` defaults to ``bundled`` if you've already installed
-   llvm in third-party and ``none`` otherwise.
-
-   Chapel currently supports LLVM 11.0.
-
-   .. note::
-
-       We have had success with this procedure to install LLVM 11.0
-       dependencies on Ubuntu.
-
-       First, follow the instructions at ``https://apt.llvm.org`` that
-       explain how to place the appropriate lines into
-       ``/etc/apt/sources.list.d/llvm-toolchain.list`` and retrieve
-       the archive signature, then do the following.
+   If unset, ``CHPL_LLVM`` defaults to ``bundled`` if you've already
+   installed llvm in third-party or ``system`` if a compatible
+   system-wide installation of LLVM is detected. If neither of those are
+   the case you will need to either add a system-wide installation of
+   LLVM or set ``CHPL_LLVM`` to ``bundled`` or ``none``.
```

Comment:
Shouldn't there be a linux32 special-case here?  (I was surprised to see it earlier in the CHPL_TARGET_COMPILER section, as I think of linux32 as being more about the default for CHPL_LLVM than CHPL_TARGET_COMPILER in an LLVM back-end world).

Reply:
Good point, I will fix.